modules/store/src/Plugin/views/argument_default/ActiveStore.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\commerce_store\Plugin\views\argument_default\ActiveStore.
+ */
```

Comment:
The @file docblocks for files that have only one class/interface can be removed, so let's remove this.